org.civicrm.bootstrapcivihr/scss/mixins/_spinner.scss

```diff
@@ -0,0 +1,39 @@
+// Spinner related variables
+$default-height: 32px;
+$default-width: 32px;
+$speed: 1s;
+
+/**
+ * Spinner mixin
+ * $speed Speed to rorate the spinner in seconds
+ * $height Hight of the spinner
+ * $height Width of the spinner
+ */
+@mixin spinner($speed: $speed, $height: $default-height, $width: $default-width) {
+  animation: spin $speed linear infinite;
+  background: url(../img/spinner.svg) no-repeat center center !important;
+  display: block;
+  margin: auto;
+
+  @if $height {
+    height: $height !important;
+  } @else {
```

Comment:
actually i don't get this if/else at all, you are defining already the default values, so what's the point of this?

Reply:
Right! thanks for catching it! in fact we won't be needing it!